docs/source/getting_started/benchmarks.rst

```diff
@@ -47,98 +47,75 @@ self-supervised models from scratch on the training data. At the end of every
-
-    Plot with the kNN accuracy on test set for training the models for 200 epochs. The shorter
-    runs (fewer steps) are the ones with batch size of 512. Green: MoCo, Gray: SimCLR, 
-    Orange: SimSiam, Red: MoCo, Light Blue: SimCLR, Pink: SimSiam.
+  :header: "Model", "Epochs", "Batch Size", "kNN test Accuracy", "Runtime", "Peak GPU consumption"
+  :widths: 20, 20, 20, 20, 20, 20
+
+  "BarlowTwins", 200, 128, 0.835, "193.4 Min", "2.2 GByte"
+  "BYOL", 200, 128, 0.872, "217.0 Min", "2.3 GByte"
+  "DINO", 200, 128, 0.868, "220.7 Min", "2.3 GByte"
+  "Moco", 200, 128, 0.838, "229.5 Min", "2.3 GByte"
+  "NNCLR", 200, 128, 0.838, "198.7 Min", "2.2 GByte"
+  "SimCLR", 200, 128, 0.822, "182.7 Min", "2.2 GByte"
+  "SimSiam", 200, 128, 0.779, "182.6 Min", "2.3 GByte"
+  "SwaV", 200, 128, 0.806, "182.4 Min", "2.2 GByte"
+  "BarlowTwins", 200, 512, 0.827, "160.7 Min", "7.5 GByte"
+  "BYOL", 200, 512, 0.872, "188.5 Min", "7.7 GByte"
+  "DINO", 200, 512, 0.862, "191.1 Min", "7.5 GByte"
+  "Moco (*)", 200, 512, 0.850, "196.8 Min", "7.8 GByte"
+  "NNCLR (*)", 200, 512, 0.836, "164.7 Min", "7.6 GByte"
+  "SimCLR", 200, 512, 0.828, "158.2 Min", "7.5 GByte"
+  "SimSiam", 200, 512, 0.814, "159.0 Min", "7.6 GByte"
+  "SwaV", 200, 512, 0.833, "158.4 Min", "7.5 GByte"
+  "BarlowTwins", 800, 512, 0.857, "641.5 Min", "7.5 GByte"
+  "BYOL", 800, 512, 0.911, "754.2 Min", "7.8 GByte"
+  "DINO", 800, 512, 0.884, "765.5 Min", "7.6 GByte"
+  "Moco (*)", 800, 512, 0.900, "787.7 Min", "7.8 GByte"
+  "NNCLR (*)", 800, 512, 0.896, "659.2 Min", "7.6 GByte"
+  "SimCLR", 800, 512, 0.875, "632.5 Min", "7.5 GByte"
+  "SimSiam", 800, 512, 0.906, "636.5 Min", "7.6 GByte"
+  "SwaV", 800, 512, 0.881, "634.9 Min", "7.5 GByte"
+
+(*): Increased size of memory bank from 4096 to 8192 to avoid too quickly 
+changing memory bank due to larger batch size.
 
 We make the following observations running the benchmark:
 
-- SimCLR and SimSiam benefit from larger batch sizes. MoCo (which uses a
-  memorybank) performs very well also for smaller batch sizes.
-- All models need around 3h (on a V100 GPU) to complete the 200 epoch benchmark 
-  and use roughly the same amount of GPU memory.
-
-Furthermore, we conducted experiments training the models for 800 epochs.
-
-We can train the models longer and will note an increase in test accuracy. 
-Self-Supervised models benefit from long training times.
-
-.. figure:: images/bench_knn_accuracy_cifar10_800epochs.png
+- Self-Supervised models benefit from larger batch sizes and longer training.
+- All models need around 3-4h to complete the 200 epoch benchmark and 11-13h
+  for the 800 epoch benchmark.
+- Memory consumption is roughly the same for all models.
+- Some models, like MoCo or SwaV, learn quickly in the beginning and then 
+  plateau. Other models, like SimSiam or NNCLR, take longer to warm up but then
+  catch up when training for 800 epochs. This is also clearly seen in the figure
```

Comment:
This can also be seen in the figure below or this is also clearly visible

Reply:
Fixed